source/fundamentals/auth.txt

```diff
@@ -0,0 +1,384 @@
+using the following placeholders:
+
+* ``username`` - your MongoDB username
+* ``password`` - your MongoDB user's password
+* ``hostname`` - network address of your MongoDB server, accessible by your client
+* ``port`` - port number of your MongoDB server
+* ``authenticationDb`` - MongoDB database that contains your user's
+  authentication data. If you omit this parameter, the driver uses the
+  default value ``admin``.
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions and sample code for specifying this authentication
+mechanism:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify the default authentication mechanism using a connection
+      string, explicitly omit the mechanism. Your code to instantiate
+      a ``MongoClient`` should resemble the following:
+
+      .. code-block:: java
+
+         MongoClient mongoClient = MongoClients.create("mongodb://<username>:<password>@<hostname>:<port>/?authSource=<authenticationDb>");
+
+   .. tab::
+      :tabid: MongoCredential
+
+      To specify the default authentication mechanism using the
+      ``MongoCredential`` class, use the ``createCredential()`` method. Your
+      code to instantiate a ``MongoClient`` should resemble the following:
+
+      .. include:: /includes/fundamentals/code-snippets/auth-credentials-default.rst
+
+For more information on the challenge-response (CR) and salted
+challenge-response authentication mechanisms (SCRAM) that MongoDB supports,
+see the :manual:`SCRAM </core/security-scram/>` section of the server manual.
+
+.. _scram-sha-256-auth-mechanism:
+
+``SCRAM-SHA-256``
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   ``SCRAM-SHA-256`` is the default authentication method for MongoDB starting
+   in MongoDB 4.0.
+
+``SCRAM-SHA-256`` is a salted challenge-response authentication mechanism
+(SCRAM) that uses your username and password, encrypted with the ``SHA-256``
+algorithm to authenticate your user.
+
+The following code snippets show how to specify the connection mechanism,
+using the following placeholders:
+
+* ``username`` - your MongoDB username
+* ``password`` - your MongoDB user's password
+* ``hostname`` - network address of your MongoDB server, accessible by your client
+* ``port`` - port number of your MongoDB server
+* ``authenticationDb`` - MongoDB database that contains your user's
+  authentication data. If you omit this parameter, the driver uses the
+  default value ``admin``.
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions and sample code for specifying this authentication
+mechanism:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify the ``SCRAM-SHA-256`` authentication mechanism using a
+      connection string, pass the ``SCRAM-SHA-256`` value as a parameter
+      of ``authMechanism`` in your connection string. Your code to instantiate
```

Comment:
"pass the ``SCRAM-SHA-256`` value as a parameter of ``authMechanism`` in your connection string" sounds strange to me, though I can't quite say why. I would probably word this more like: "assign the ``authMechanism`` parameter a value of ``SCRAM-SHA-256`` in your connection string. What do you think? This applies to all of the connection string tabs across the page.

Reply:
Great suggestion, will update.